### PR TITLE
Small improvements for onboarding experience

### PR DESCRIPTION
**Pull request by trkohler**
connect: https://github.com/the-guild-org/conductor/issues/566

YAML config didn't work from the freshest master, so while it's in progress - I thought that json example recommendation would work better

### CODE REVIEW

**Comment by YassinEldeeb** [comment]
hey, @trkohler. thanks for the PR! As mentioned [here](https://github.com/the-guild-org/conductor/issues/566#issuecomment-2056942645), loading a yaml config file works, but it needs to be supplied as an argument, as it defaults to `./config.json` when not specified.

In the readme, we have that argument supplied when yaml is used, as the following:
```sh
npx @graphql-conductor/bin ./conductor.config.yaml
```

So, I'll close this, thank you! 🙏

**Comment by YassinEldeeb** [comment]
I think it's still valuable to include an example of a JSON config, let's merge this!

File: README.md [hunk 1]
```diff
@@ -52,27 +52,41 @@ Conductor's configuration can be defined in both YAML and JSON formats. The conf
...
-      endpoint: https://my-source.com/graphql
-
-endpoints:
-  - path: /graphql
-    from: my-source
-    plugins:
-      - type: cors
-        config:
-          allowed_origin: "*"
-      - type: graphiql
```
Comment (YassinEldeeb) [inline README.md:75]: how about, we have to sections like this? 👀

### Configuration File Example (YAML)


```yaml
server:
  port: 9000

logger:
  filter: error

sources:
  - type: graphql
    id: my-source
    config:
      endpoint: https://my-source.com/graphql

endpoints:
  - path: /graphql
    from: my-source
    plugins:
      - type: cors
        config:
          allowed_origin: "*"
      - type: graphiql
```


### Configuration File Example (JSON)

```json
{
  "server": {
    "port": 9000
  },
  "logger": {
    "filter": "error"
  },
  "sources": [
    {
      "type": "graphql",
      "id": "my-source",
      "config": {
        "endpoint": "https://my-source.com/graphql"
      }
    }
  ],
  "endpoints": [
    {
      "path": "/graphql",
      "from": "my-source",
      "plugins": [
        {
          "type": "cors",
          "config": {
            "allowed_origin": "*"
          }
        },
        {
          "type": "graphiql"
        }
      ]
    }
  ]
}
```

File: README.md [hunk 2]
```diff
@@ -83,6 +97,12 @@ Conductor can be ran via the docker image, and it can even be ran via `npx` for
 npx @graphql-conductor/bin ./conductor.config.yaml
 ```
 
+Locally:
+  
+```sh
+cargo run --bin conductor
+```
+
```
Comment (YassinEldeeb) [inline README.md:105]:
```suggestion
Or, locally:
  
```sh
cargo run --bin conductor ./conductor.config.json
```

```

File: README.md [hunk 3]
```diff
@@ -83,6 +97,12 @@ Conductor can be ran via the docker image, and it can even be ran via `npx` for
 npx @graphql-conductor/bin ./conductor.config.yaml
```
Comment (YassinEldeeb) [inline README.md:97]: maybe we should also point out above that `config.json` is the default one, and it should be supplied as a first argument otherwise.

Comment (trkohler) [reply README.md:122]: added there: https://github.com/the-guild-org/conductor/pull/567/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R131

File: README.md (hunk 1 above)
Comment (trkohler) [reply README.md]: did

**Pull request merged**